### DISCUSSION TITLE
Recommended server specs?

### DISCUSSION THREAD

**Discussion by ethndotsh** [quirrel-dev/quirrel#1101] [General]
Hello! I am planning to use Quirrel but wanted to know how much system resources it uses for low/medium usage, and what the recommended system resources are.

**Comment by Skn0tt** [comment]
Hi @cursecodes! I have not performed any experiments around what system resource configs work best, but especially for low/medium usage you won't need much. The hosted version of Quirrel I was running ran on Heroku's Premium-0 Redis and Standard-1X Dynos, which were the smallest available sizes.

**Reply by ethndotsh** [reply]
Was it a new Standard-1X for each user, or a handful for the entire service?

For roughly 20000 events/mo, would a single Basic Heroku dyno be able to keep up? What would be the limit for a single Basic Heroku dyno in your opinion for events/mo?

**Reply by Skn0tt** [reply]
It was one Standard-1x for the entire service.

**Reply by ethndotsh** [reply]
Woah! Those must've been pretty good margins then.

I've edited my post above, please take a look.

**Reply by Skn0tt** [reply]
> For roughly 20000 events/mo, would a single Basic Heroku dyno be able to keep up? What would be the limit for a single Basic Heroku dyno in your opinion for events/mo?

As I've said, I haven't performed any testing as to what the limit would be. 20k per month should definitely work, assuming it's somewhat even-spread. (20k per month is one every two minutes!)

**Reply by ethndotsh** [reply]
A better question would be how many events/mo were you managing with the one Standard-1X? I'm very curious.
